src/azure-cli/azure/cli/command_modules/keyvault/vendored_sdks/azure_keyvault_securitydomain/_serialization.py

```diff
@@ -53,12 +53,13 @@
     List,
 )
 
-try:
-    from urllib import quote  # type: ignore
-except ImportError:
-    from urllib.parse import quote
+from urllib.parse import quote
 import xml.etree.ElementTree as ET
 
+from datetime import timezone as _FixedOffset  # type: ignore
```

Comment:
I'll fix this. The SDK has this serialization file duplicated in quite a few directories.